pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,228 @@
-                            </td>
-                            <% } %>
-                        </tr>
-                        <% }); %>
-                    </tbody>
-                </table>
+            <div class="col-auto">
+              <% if ((authz_data.has_course_permission_edit) && (! course.example_course) && (! locals.needToSync)) { %>
+              <form class="d-inline" name="add-question-form" method="POST">
+                <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                <button name="__action" value="add_question" class="btn btn-sm btn-light"><i class="fa fa-plus" aria-hidden="true"></i>
+                  <span>Add question</span></button>
+              </form>
+              <% } %>
             </div>
-            <script>
-                $(function() {
-                    $("#questionsTable").tablesorter({
-                        theme: "bootstrap",
-                        widthFixed: true,
-                        headerTemplate: '{content} {icon}',
-                        widgets: ["uitheme", "filter", "zebra"],
-                        widgetOptions: {
-                            zebra: ["even", "odd"],
-                            filter_reset: "#resetFilter",
-                            filter_cssFilter: "form-control",
-                            filter_functions: {
-                                3: {
-                                    <% all_tags.forEach(function(tag) { %>
-                                    "<%= escapeRegExp(tag.name) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(tag.name) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% if (locals.course_instance) { %>
-                                5: {
-                                    <% all_assessments.forEach(function(assessment) { %>
-                                    "<%= escapeRegExp(assessment.label) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(assessment.label) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% } %>
-                            },
-                        },
-                    });
-                });
-            </script>
+          </div>
+        </div>
+
+        <table id="questionsTable"
+               data-data="<%= JSON.stringify(questions) %>"
+               data-classes="table table-sm table-hover table-bordered"
+               data-thead-classes="thead-light"
+               data-filter-control="true"
+               data-show-filter-control-switch="true"
+               data-show-search-clear-button="true"
```

Comment:
Assuming this is the button below, I'm a bit worried that users won't know what "filters" are, and the trash icon looks scary. Could we maybe label it something like "Show all questions" or "Show everything"? With a friendlier icon of some kind? 😄

<img width="136" alt="image" src="https://user-images.githubusercontent.com/875833/128927873-08ed4a50-4bbd-4adb-840d-ac17228f4399.png">

Reply:
It turns out that I dislike "Show all questions" even more, sorry! I actually clicked it thinking that it was going to remove pagination 😄

I guess I think we should go back to "Clear filters". Perhaps with a generic X icon (or no icon) rather than the trash?